PaddleCV/object_detection/README.md

```diff
@@ -75,31 +76,38 @@ python tools/infer.py -c configs/mask_rcnn_r50_1x.yml \
     -infer_img=demo/000000523957.jpg
 ```
 
-The predicted  and visualized results are saved in `output/1.jpg`.
+For detailed training and evaluation workflow, please refer to [GETTING_STARTED.md](docs/GETTING_STARTED.md).
 
-For more detailed training and evaluating pipeline, please refer [GETTING_STARTED.md](docs/GETTING_STARTED.md).
+We also recommend users to take a look at the [IPython Notebook demo](demo/mask_rcnn_demo.ipynb)
 
+Further information can be found in these documentations:
 
-More documentation:
+- [Introduction to the configuration workflow.](docs/CONFIG.md)
+- [Guide to custom dataset and preprocess pipeline.](docs/DATA.md)
 
-- [How to config an object detection pipeline.](docs/CONFIG.md)
-- [How to use customized dataset and add data preprocessing.](docs/DATA.md)
 
+##  Todo List
 
-## Deploy
+Please note this is a work in progress, substantial changes may come in the
+near future.
+Some of the planned features include:
 
-The example of how to use PaddlePaddle to deploy detection model will be added later.
+- [ ] Mixed precision training.
+- [ ] Distributed training.
+- [ ] Inference in 8-bit mode.
+- [ ] User defined operations.
+- [ ] Larger model zoo.
 
-## Updates
 
-The major updates are as follows:
+## Updates
 
-#### 2019-07-03
-- Release the unified detection framework.
-- Supported algorithms: Faster R-CNN, Mask R-CNN, Faster R-CNN + FPN, Mask R-CNN + FPN, Cascade-Faster-RCNN + FPN, RetinaNet, Yolo v3, and SSD.
-- Release the first version of Model Zoo.
+#### v0.1 (7/3/2019)
```

Comment:
Do not add v0.1? Since there is no version num now.

Reply:
then what should I put there?

Reply:
changed to "initial relsase"